R/tt_as_df.R

```diff
@@ -101,6 +107,41 @@ as_result_df <- function(tt, spec = NULL,
           cellvals <- mf_result_numeric
         }
       }
+
+      diff_in_cellvals <- length(unlist(cellvals_init)) - length(unlist(cellvals))
+      if (make_ard && abs(diff_in_cellvals) > 0) {
+        warning_msg <- paste0(
+          "We found ", abs(diff_in_cellvals), " values that have cell values that differ from ",
```

Comment:
Suggestion for conciseness
```suggestion
          "Found ", abs(diff_in_cellvals), " cell values that differ from ",
```